modules/lib/lib-vhost/src/main/resources/lib/xp/vhost.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Functions to find virtual host.
+ *
+ * @example
+ * var vhostLib = require('/lib/xp/vhost');
+ *
+ * @module content
+ */
+
+function checkRequiredValue(value, name) {
+    if (value === undefined) {
+        throw 'Parameter \'' + name + '\' is required';
+    }
+}
+
+exports.isEnabled = function () {
+    const bean = __.newBean('com.enonic.xp.lib.vhost.VirtualHostHandler');
+    return __.toNativeObject(bean.isEnabled());
```

Comment:
Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6210598)